packages/client/src/scenes/Home.tsx

```diff
@@ -381,20 +398,21 @@ export default class Home extends Component<IProps, IState> {
             );
         }
 
-        return rooms.map(({ roomId, metadata, clients, maxClients }, index) => {
-            const map = MapsList.find((item) => item.value === metadata.roomMap);
-            const mapName = map ? map.title : metadata.roomMap;
+        return rooms.map(({ initialConfig, roomId, state }, index) => {
+            const typedState = state as TypeLobbyState;
+            const typedInitialConfig = initialConfig as TypeLobbyInitialConfig;
+            const clientCount = typedState?.playerCount ?? 0;
 
             return (
                 <Fragment key={roomId}>
                     <Room
                         id={roomId}
-                        roomName={metadata.roomName}
-                        roomMap={mapName}
-                        clients={clients}
-                        maxClients={maxClients}
-                        mode={metadata.mode}
-                        onClick={this.handleRoomClick}
+                        roomName={typedInitialConfig.roomName}
+                        roomMap={typedInitialConfig.mapName}
+                        clients={clientCount}
```

Comment:
```suggestion
                        clients={typedState?.playerCount ?? 0}
```
and then we can get rid of the `const clientCount` line above